### ISSUE TITLE
[Accessibility]: Chart values are not screen readable ( with JAW tool )

### ISSUE THREAD

**Issue by tushar-dhumal** [carbon-design-system/carbon-charts#1963]
### Application/Team

GKLM

### Environment

Browser

### What happened? What did you expect to happen instead?

We have encountered an accessibility issue with our bar and pie charts while using the JAWS screen reader. When navigating through the charts using the Tab key in any browser, the screen reader does not announce the chart values.

Issue Details:

    Components We are using: Bar and Pie Charts
    Screen Reader: JAWS
    Issue Description: When navigating via Tab, the screen reader does not read out the chart values, making it inaccessible for visually impaired users.
    Browsers Tested:  Chrome, Firefox

Expected Behavior:
When a user navigates to the chart using Tab, the screen reader should read out the relevant chart data (e.g., axis labels, values, and categories).

Could you please investigate and suggest possible solutions or fixes for this issue? Let us know if any additional details are needed.

### What WCAG 2.1 checkpoint does the issue violate?

1.1.1, 1.3.1, 1.4.3

### Version

"@carbon/charts-react": "^1.18.0",

### Data & options used

```js

```

### Relevant log output

```js

```

### What priority level would this issue be in your opinion?

P0

**Comment by theiliad** [comment]
Hi,
We’ve done various rounds of accessibility analysis and design around our charts.

Our accessibility designer at the time was of the opinion that keyboard navigation through charts is not a great idea, since it doesn’t actually paint a mental picture of what the chart is visualizing (e.g. trends, patterns etc…)

That is why the tabular view was designed in charts

**Comment by steven1046** [comment]
@theiliad is there currently a keyboard sequence that can tab to thru the table in tabular view? I see tabindex of -1 on the whole modal and it tabs to the x button but not the table/values.
